MakeHist/cutflow.json

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
This file is a result of code execution. It should not be present in the repository. Please remove it and provide instead a description of cut-flow JSON file structure,